apps/condo/bin/discover-service-consumers.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Update address by id with new data got from external provider.
+ * Use .env to detect particular provider.
+ *
+ * Usage:
+ *      yarn workspace @app/condo node bin/discover-service-consumers [--dry-run] "some address" unitType unitName [billingAccountId [residentId]]
+ */
+
+const path = require('path')
+
+const get = require('lodash/get')
+
+const { prepareKeystoneExpressApp } = require('@open-condo/keystone/test.utils')
+
+const { discoverServiceConsumers } = require('@condo/domains/resident/utils/serverSchema')
+
+const dv = 1
+const sender = { dv, fingerprint: 'discover-service-consumers-script' }
+
+async function main (args) {
```

Comment:
Why do you need this script?

There's no chance will be using it N times for each unitName :)